pyvisa_py/protocols/rpc.py

```diff
@@ -454,7 +454,8 @@ class RawTCPClient(Client):
 
     def __init__(self, host, prog, vers, port, open_timeout=5000):
         Client.__init__(self, host, prog, vers, port)
-        self.connect((open_timeout / 1000.0) + 1.0)
+        open_timeout = open_timeout or 5000
```

Comment:
Avoid turning a 0 into 5000
```suggestion
        open_timeout = open_timeout if open_timeout is not None else 5000
```